src/components/header/menu/dropdowns/MenuDropdownFile.vue

```diff
@@ -69,6 +75,19 @@ export default {
         });
     },
     methods: {
+        newDrawing () {
+            this.$instances.modeler = new Modeler();
+            this.$instances.modeler.addFormalism(PluginBase);
+            this.$instances.modeler.addFormalism(PluginPT);
+            this.$instances.simulator = new Simulator();
+            this.$instances.simulator.addFormalism(PluginBase);
+            this.$instances.simulator.addFormalism(PluginPT);
+
+            this.$store.commit('setActiveContext');
+            setTimeout(() => {
```

Comment:
We should eventually find a way that avoids the need to do this, but fine for now :smile:

Reply:
Sry for this, but I couldn't figure it out. :( Where could we put it, so we dont forget about it? An own issue, or an existing one?

Reply:
Maybe I leave #13 open.

Reply:
Can you create a new issue for this? Maybe describe why you couldn't figure it out / what else you have already tried?

Reply:
I've reopend the issue already. :D

Reply:
But, you're right and move it to this new issue #27.